support-workers/cloud-formation/src/templates/cfn-template.yaml

```diff
@@ -288,8 +287,8 @@ Resources:
           Value: !Ref Stage
       ComparisonOperator: LessThanOrEqualToThreshold
       Threshold: 0
-      Period: 300
-      EvaluationPeriods: 36
+      Period: 3600
```

Comment:
I changed them all to 3600s interval (1 hour) so you can tell how long they are at a glance